tasks/set_vars.yml

```diff
@@ -16,3 +16,23 @@
   vars:
     __mssql_vars_file: "{{ role_path }}/vars/{{ item }}"
   when: __mssql_vars_file is file
+
+- name: Determine if system is booted with systemd
+  when: not __mssql_is_booted is defined
+  block:
+    - name: Run systemctl
+      # noqa command-instead-of-module
+      command: systemctl is-system-running
+      register: __is_system_running
+      changed_when: false
+      failed_when: false
+
+    - name: Require installed systemd
+      fail:
+        msg: "Error: This role requires systemd to be installed."
+      when: __is_system_running.msg is defined and "No such file or directory" in __is_system_running.msg
```

Comment:
```suggestion
      when: '"No such file or directory" in __is_system_running.msg | d("")'
```

Reply:
Ah nice, thanks! I'll apply that to the existing roles together with the "degraded" fix.